enterprise/causal-clustering/src/test/java/org/neo4j/causalclustering/discovery/Cluster.java

```diff
@@ -226,21 +226,28 @@ public void shutdown()
     {
         try ( ErrorHandler errorHandler = new ErrorHandler( "Error when trying to shutdown cluster" ) )
         {
-            shutdownCoreMembers( errorHandler );
+            shutdownCoreMembers( errorHandler, coreMembers() );
             shutdownReadReplicas( errorHandler );
         }
     }
 
-    private void shutdownCoreMembers( ErrorHandler errorHandler )
+    private void shutdownCoreMembers( ErrorHandler errorHandler, Collection<CoreClusterMember> members )
     {
-        shutdownMembers( coreMembers(), errorHandler );
+        shutdownMembers( members, errorHandler );
     }
 
     public void shutdownCoreMembers()
     {
+        shutdownCoreMembers( coreMembers().toArray( new CoreClusterMember[0] ) );
+    }
+
+    public void shutdownCoreMembers( CoreClusterMember... members )
+    {
+        List<CoreClusterMember> membersColl = Arrays.asList( members );
```

Comment:
If we call `shutdownCoreMembers` then we convert the core members to an array to pass it in here, and then convert it back into a list.